### ISSUE TITLE
Vortex stuck at "Loading Extensions"

### ISSUE THREAD

**Issue by joshvickerson** [Nexus-Mods/Vortex#7670]
**Describe the bug**
When opening Vortex, it never progresses past the "Loading Extensions" screen and just hangs

**To Reproduce**
1. Update to Vortex 1.3.13 (when this started for me)
2. Open Vortex

**Expected behavior**
I'd expect it to continue loading as normal.

**Screenshots**
![vortex_stuck](https://user-images.githubusercontent.com/2325487/95757025-9629e600-0c74-11eb-82b7-63f72bbbcac1.png)

**Platform (please complete the following information):**
 - OS: Windows 10 [ver 2004]
 - Vortex Version 1.3.13

**Comment by TanninOne** [comment]
please provide vortex.log and vortex1.log (found at C:\Users\<username>\AppData\Roaming\Vortex)

**Comment by joshvickerson** [comment]
Sure thing, here ya go!

**Comment by TanninOne** [comment]
Do you know if you enabled the option "Allow Symlinks without elevation" under Settings->Workarounds?

For that feature Vortex has to create a windows task and it seems to hang while communicating with windows about creating or checking for the existence of that task.

If you're comfortable using the command line we can force the option to be off.
For that, close Vortext, then open a command prompt (cmd.exe) and enter
`cd "C:\Program Files\Black Tree Gaming Ltd\Vortex"`
then
`Vortex.exe --set settings.workarounds.userSymlinks=false`

then you can try to restart Vortex.

If this works and you still want to use the feature you can try re-enabling it once Vortex is up, maybe recreating the task is all it needs, if it fails again you can use the same steps to disable it again.

Either way, please let us know how it goes.

**Comment by joshvickerson** [comment]
I don't recall if I had set that or not, but when I ran the command as suggested it reported "changed". Would it only report that if the setting did actually change (i.e. if it had been true), or does it report "changed" just if it completes successfully?

I'm not sure when or why I would have changed that setting so I can leave it disabled for now. If it's helpful for debugging purposes to provide fresh logs, or re-enable it and provide logs let me know.

**Comment by TanninOne** [comment]
You didn't say whether Vortex starts up now.
But no, unfortunately it would always say "changed" as long as you provide a valid setting, even if it's identical to the old.

**Comment by joshvickerson** [comment]
Apologies! Vortex does start up now with that setting disabled.

**Comment by TanninOne** [comment]
Great to hear it, that narrows down the problem a lot. Thank you for getting back to us.

**Comment by TanninOne** [comment]
I'm fairly certain this was fixed by commit da2d641b0fe995f86ba638f63741865bb079779a

**Comment by joshvickerson** [comment]
@TanninOne this issue has returned for me (currently on latest Vortex stable). I did re-enable the symlink without elevation feature as part of some debugging when I had issue deploying mods for Stardew Valley. Vortex continued to launch successfully for a while even with that setting still enabled. Today it got stuck on Loading Extentions when it opened while handling a download link from Nexus. The previous workaround you suggested for the symlink setting didn't allow Vortex to open this time.

Attached is my log file. Let me know if there's anything else I can do to help track down the cause of this issue!

**Comment by joshvickerson** [comment]
Update: After restarting my computer I was able to open Vortex and I had this notification about the symlink workaround not being fully disabled.

![image](https://user-images.githubusercontent.com/2325487/107128518-54a8f080-688c-11eb-85ec-aadf74fff2eb.png)

**Comment by yomice1234** [comment]
ive been having the same problem but on a updated version of vortex

**Comment by LoliGagging** [comment]
I had the same issue this same fix fixed it for me. I did not get the notification about re-enabling symlink after vortex restarted. Thank you!

I'm not sure if it is related but I was using this for starfield before this issue occurred with vortex. sfse_loader used to launch stafield with mods took a very long time to load and so did the game as a result. This was after opening and updating vortex and one mod for the first time since I installed all 28 mods 3 weeks ago.
